config/manager/kustomization.yaml

```diff
@@ -11,8 +11,3 @@ configMapGenerator:
 - files:
   - controller_manager_config.yaml
   name: manager-config
-
-images:
```

Comment:
Setting the image of the controller now happens in the bundle itself so that it can use the image distributed with the bundle.